bundle/metadata/annotations.yaml

```diff
@@ -4,8 +4,7 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: aws-nlb-helper-operator
-  operators.operatorframework.io.bundle.channels.v1: alpha,stable
-  operators.operatorframework.io.bundle.channel.default.v1: alpha
+  operators.operatorframework.io.bundle.channels.v1: alpha
```

Comment:
If it is an **alpha** release, why it is `0.1.1` instead of something like `0.1.1-alpha-X`?

Reply:
Fixed in https://github.com/3scale-ops/aws-nlb-helper-operator/pull/20/commits/b92570fa1c88fde6bd7deab03f928283c07b4ac4